client-js/connection.js

```diff
@@ -42,7 +42,7 @@ module.exports = function () {
         renderTesting();
         $.ajax({
             type: "POST",
-            url: "/connections/test",
+            url: baseUrl + "/sqlpad/connections/test",
```

Comment:
missed this with my initial testing. looks like /sqlpad snuck in during initial development